webui/react/src/hooks/useModal/HyperparameterSearch/useModalHyperparameterSearch.tsx

```diff
@@ -446,13 +447,12 @@ const useModalHyperparameterSearch = ({
             </SelectFilter>
           </Form.Item>
           <Form.Item
-            initialValue={experiment.configRaw?.resources?.slots_per_trial ??
-              maxSlots === 0 ? 0 : 1}
```

Comment:
this ternary should have parentheses around it, as it's evaluating to `0` whenever `maxSlots` is 0, even when `slots_per_trial` is defined as greater than 0.

Reply:
talking to backend, this value should never be 0, which causes scheduling problems.